src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1287,3 +1373,14 @@ def load_pipeline_from_original_stable_diffusion_ckpt(
         pipe = LDMTextToImagePipeline(vqvae=vae, bert=text_model, tokenizer=tokenizer, unet=unet, scheduler=scheduler)
 
     return pipe
+
+
+class FromCkptMixin:
+    @classmethod
+    def from_ckpt(cls, model_path_or_checkpoint, **kwargs):
+        # silences transformers warning on CLIP Model loading unused keys
+
+        logging.set_verbosity_error()
```

Comment:
let's not silence logs in a class method like this. That should be the job of the user if they want to. Especially as `logging.set_verbosity_error` is mutative and will permanently change the logging verbosity for the rest of the process

Reply:
Okay, makes sense! I've removed the logging setting in the latest commit